packages/rolldown/src/log/logs.ts

```diff
@@ -20,8 +21,9 @@ export function logParseError(message: string): RollupLog {
 export function logMinifyWarning(): RollupLog {
   return {
     code: MINIFY_WARNING,
-    message:
+    message: colors.yellow(
       'The built-in minifier is still under development. Setting "minify: true" is not recommended for production use.',
+    ),
```

Comment:
Could we add the color the  handler of `warning`? Look like we are not handling the warning at cli.

Reply:
Since `RollupLog` doesn't have `Severity` we don't even know which Log should render as `warning`(yellow) or `error`(red) or other color. Let's add `Severity` first